src/main/java/com/example/App.java

```diff
@@ -0,0 +1,16 @@
+package com.example;
+
+import java.io.IOException;
+
+public class App {
+    public App() throws IOException {
+//        super();
+    }
+    public static void main(String[] args) {
+        try {
+            new App();
+        } catch (IOException ioe) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://docker.for.mac.localhost:9000/coding_rules#rule_key=squid%3AS00108)